### ISSUE TITLE
SwinIR: 109 Dropout layers with drop_rate=0

### ISSUE THREAD

**Issue by jxtps** [cszn/KAIR#104]
The default car_jpeg SwinIR model has 109 dropout layers, all with a drop_rate of 0. 

Shouldn't it either use some reasonable non-zero drop_rate, or omit the Dropout layers (which will presumably add to training & inference time)? 

Did you experiment with non-zero drop_rates?

**Comment by JingyunLiang** [comment]
Most restoration networks don't use dropout layer (i.e., dropout rate =0). I haven't tested it with non-zero values. I think it won't have large impact on the performance.

**Comment by jxtps** [comment]
Ok. I tried removing them but it didn't seem to impact training speed, so I guess pytorch is clever enough under the hood to check for p=0 ;)
